_inc/client/components/masthead/style.scss

```diff
@@ -1,25 +1,29 @@
 .jp-masthead {
 	background-color: $green-secondary;
 	text-align: center;
+
+	@media (max-width: rem( 720px ) ) {
```

Comment:
Oh hah, I meant the one next to `max-width`. From a quick look around, it looks like we do actually normally have a space after `media`, and then no space after its `(`. Weird :)

We normally don't have a space either side of the content of the `media` parens though, which would mean the last two should be `))` (no space), but now I just feel like a pedant, especially since that's harder to read... so... probably ignore all of this.